site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -2,370 +2,107 @@
-```
+### Other assets {: #assets }
 
-You can find the <code><var>EXTENSION_ID</var></code> in the Extension management page
-**chrome://extensions**. The <code><var>RELATIVE_PATH</var></code> is the file path relative to the
-extension's top folder.
+An extension can include many types of resources, but only the [extension icons][manifest-icons] are required. All assets, including images and fonts, must be part of the extension package.
 
-{% Aside 'key-term' %}
+## How they work together {: #interact }
 
-The **extension ID** is a 32-character alpha string that identifies an extension in the browser and
-on the Chrome Web Store.
+In this section, we will describe how these extension components communicate, store data, and share access to resources.
 
-{% endAside %}
+### Sending messages {: #pageComm }
 
-During development, a new ID is generated when an [_unpacked extension_][docs-unpacked] is loaded,
-unless the `"key"` property is [set in the manifest][docs-key].
+Many times content scripts, or other extension pages, need to send or receive information from the extension service worker. In these cases, either side can listen for messages sent from the other end, and respond on the same channel. Extensions can send a one-time request or establish a long-lived connection to support multiple messages.
 
-{% Aside 'caution' %}
+See [Message passing][doc-messages] for more details.
 
-All assets that content scripts and websites want to access must be declared under
-[`web_accessible_resources`][section-web-res] key in the manifest.
+### Storing data {: #data }
 
-{% endAside %}
+Chrome provides extensions with an optimized [Storage API][api-storage] available to all extension components. It includes four separate storage areas for specific use cases and an event listener that tracks whenever data is updated. For example, when you save changes in the popup, the extension service worker can respond with specified logic.
 
-### Web-accesible resources {: #web-resources }
+See [Storage API][api-storage] for usage and code samples.
 
-Web-accessible resources are files (images, HTML, CSS, Javascript) inside an extension that can be
-accessed by a content script, web pages, or other extensions. 
+### Referencing extension resources {: #ref-files }
 
-You can declare which resources are exposed and to what origins in the
-manifest:
+Extension HTML pages can use the same tags as a regular HTML page to add an extension asset, but all resources must be included in the extension bundle. Content Scripts can also access extension resources but require some extra steps described in [Accessing extension files in Content Scripts][doc-ref].
 
-```json
-{
-  ...
-  "web_accessible_resources": [
-    {
-      "resources": [ "images/*.png" ],
-      "matches": [ "https://example.com/*" ]
-    }
-  ],
-  ...
-}
-```
-
-See [Web-accesible resources][docs-web-acc-res] for usage information.
-
-## Using Chrome APIs {: #apis }
```

Comment:
The Chrome API section will be moved to [Reference API](https://developer.chrome.com/docs/extensions/reference/) issue #[4340](https://github.com/GoogleChrome/developer.chrome.com/issues/4340)

This will allow developers to find implementation details about the Chrome APIs in the main [reference docs](https://developer.chrome.com/docs/extensions/reference/)